builds/azure-pipelines/build-release.yml

```diff
@@ -0,0 +1,40 @@
+trigger: none
+
+pr: none
+
+schedules:
+  - cron: "0 0 * * Mon-Fri"
+    displayName: Mon-Fri at Midnight
+    branches:
+      include:
+        - dev
+
+variables:
+  solution: '**/*.sln'
+  configuration: 'Release'
+  nugetVersion: '0.1.0-preview'
+
+stages:
+- stage: BuildPublish
+  displayName: 'Release Build and Publish'
+
+  jobs:
+  - job: BuildTestPublish
+    displayName: 'Release: Build, Test and Publish on Windows'
+
+    pool:
+      vmImage: 'windows-latest'
```

Comment:
So am I understanding correctly that for build and test - we are running on all three plaforms and running tests on all three but for nuget publish we only need one since the end result does not dependend on build platform - right?

Reply:
Correct :)

Reply:
Just reading its trying to call the tests too right - then may be add the matrix here too...??

Reply:
I copied the template from the OpenAPI extension and they don't have it. My understanding is since we already have PR validation running on all 3 platforms, changes that are checked in are gated in a sense so we don't need to run all tests again in the nightly.

Reply:
Hmm... I was thinking for our pielines they all run for dalily. I am okay starting wih just this and then moving to more as needed.